en/mkdocs.yml

```diff
@@ -1004,7 +1001,6 @@ plugins:
               'develop/hot-deployment.md': 'https://mi.docs.wso2.com/en/4.3.0/develop/using-remote-micro-integrator'
               'develop/using-wire-logs.md': 'https://mi.docs.wso2.com/en/4.3.0/develop/debugging-mediation'
               'install-and-setup/setup/deployment/mi-cicd-k8s.md': 'https://mi.docs.wso2.com/en/4.3.0/install-and-setup/setup/deployment/integration-cicd-overview'
-              'mi-analytics/setting-up-mi-analytics.md': 'https://mi.docs.wso2.com/en/4.3.0/observe-and-manage/observe-overview'
```

Comment:
Let's add redirects to Analytics overview page

Reply:
Fixed with [2133075](https://github.com/wso2/docs-mi/pull/948/commits/21330756e327fd8b4780e1b520d61a6edaf4b221)